esp/bindings/SOAP/xpp/xpp/StartTag.h

```diff
@@ -39,13 +39,28 @@ namespace xpp {
   public:
     StartTag() { init(); }
 
+    StartTag(const StartTag& src) {
+      *this = src;
+    }
+
     ~StartTag() {
       if(attArr != NULL) {
         delete [] attArr;
       }
       attArr = NULL;
     }
 
+    StartTag& operator = (const StartTag& src) {
```

Comment:
I understand why you added this.  It would be nice if you could have another way of creating your shallow StartTag copy.  This operator= could easily be used by accident by someone who didn't understand the implications.  So, same thing, but in function that is less likely to be misused.

Reply:
This doesn't enable something that would otherwise be impossible. It explicitly makes something that has always been possible safe by deep-copying `attArr`.  Are you asking for an accidental copying to crash the system, which is what happens without this operator?

Reply:
Do we use the existing copy anywhere?  I'd add a private operator= to prevent it's use and avoid confusion.  But if we use it and this is a slight improvement it might be ok.

Reply:
The parser reads start tags before returning from `next`. `readStartTag` can only be called once before the next call to `next`. Assignment is used to update the output parameter with the previously cached data.

Reply:
Ok sounds like an improvement then.